packages/css/stories/Button.js

```diff
@@ -0,0 +1,19 @@
+import '../src/components/button.css';
+
```

Comment:
There should be theme CSS file imported for the whole storybook, similar to how end users would use the library